File: programmers/javascript/level1/나누어떨어지는숫자배열.js
// const arr = [5, 9, 7, 10];
// const divisor = 5;
const arr = [2, 36, 1, 3];
const divisor = 1;

function solution(arr, divisor) {
  arr = arr.filter((val) => val % divisor === 0);
  arr.sort((a, b) => {
    return a - b;
  });
  if (arr.length === 0) {
    arr.push(-1);
  }
  return arr;
}

console.log(solution(arr, divisor));

File: python/programmers/level1/나누어떨어지는숫자배열.py
def solution(arr, divisor):
    answer = []
    for x in arr:
        if x % divisor == 0:
            answer.append(x)
    answer.sort()
    if len(answer) == 0:
        answer.append(-1)
    return answer


print(solution([5, 9, 7, 10], 5))
